src/GraphQl/Queries/fundQueries.ts

```
@@ -1,87 +1,77 @@
+            }
+            createdAt
+          }
+        }
       }
     }
   }
 `;
 
+/**
+ * Query to fetch a specific fund by its ID, along with its associated campaigns.
+ * @params id - The ID of the fund campaign to be fetched.
```

ESLint

[error] tsdoc-undefined-tag: The TSDoc tag "@params" is not defined in this configuration

(tsdoc/syntax)